core/client/fs/src/main/java/alluxio/client/file/cache/evictor/FIFOEvictor.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.cache.evictor;
+
+import alluxio.client.file.cache.CacheEvictor;
+import alluxio.client.file.cache.PageId;
+import alluxio.conf.AlluxioConfiguration;
+
+import java.util.LinkedList;
+
+import javax.annotation.Nullable;
+
+/**
+ * Implementation of Evictor using FIFO eviction policy for the test.
+ */
+public class FIFOEvictor implements CacheEvictor {
```

Comment:
If this isn't intended for normal use, should we put it in the `test` package?

Reply:
remove the comment about tests -- i think this can be useful for normal use.